common/src/Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet/tests/Default/SchemaValidationTests.cs

```diff
@@ -8,24 +8,64 @@ namespace Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet.Tests.Default {
     using System.IO;
     using System.Linq;
     using System.Text;
+    using System.Text.Json;
     using Xunit;
 
     /// <summary>
-    /// These tests are designed to exercise the 
-    /// generated schema for configuration file 
-    /// validation. These tests are additionally 
-    /// found in the schema generator tool located
+    /// These tests are designed to exercise the generated schema for configuration file 
+    /// validation. These tests are additionally found in the schema generator tool located
     /// at: https://github.com/WilliamBerryiii/opcpublisherschemavalidator
+    /// 
+    /// The referenced schema file is a linked asset in the project file set to 
+    /// copy to the output build directory so that it can be easily referenced here.
     /// </summary>
     public class SchemaValidationTests {
 
+        // allow training commas in the test parsers as the main serializer for 
+        // this solution supports training commas. 
+        private static JsonDocumentOptions parseOptions = new JsonDocumentOptions {
```

Comment:
Is this still needed?

Reply:
Technically yes. There are examples of training commas in the existing unit tests and I didn't want to break too much.

Reply:
Actually, I'm on the fence here. Technically ECMA 5+ allows trailing commas in objects, though it's still theoretically invalid JSON. My hunch is that since folks may be using JS/TS to generate the config files based on OPC UA IM's, we would probably solve some pain in allowing the use of trailing commas in the config files. 

We could clean up the unit tests, given how much we've already changed; but, I hesitate to do so for fear of breaking existing users that may depend on the trailing commas

Reply:
Other reason I wanted to check this was because "parseOptions" is not being used in the test code (SchemaValidationTests.cs), guess you wanted to use it is somewhere in that class? We already have this option in JsonSchemaDotNetSchemaValidator to parse json.